src/kafka_avro_confluent/v2/serde.clj

```diff
@@ -0,0 +1,10 @@
+(ns kafka-avro-confluent.v2.serde
+  (:require [kafka-avro-confluent.v2.deserializer :as kac.des]
+            [kafka-avro-confluent.v2.serializer :as kac.ser])
+  (:import org.apache.kafka.common.serialization.Serdes))
+
+(defn ->avro-serde
+  [config & {:keys [key?]
+             :or   {key? false}}]
+  (Serdes/serdeFrom (kac.ser/->avro-serializer config :key? key?)
```

Comment:
Should the 2nd param to `->avro-serializer` be `:key` instead of `:key?` ? Just looking at the `:pre` clause in `serializers.clj`